### PR TITLE
Some updates :)

### PR DESCRIPTION
**Pull request by HideakiAtsuyo**

### CODE REVIEW

**Comment by HideakiAtsuyo** [comment]
Need:
- Create dev branch
- Set dev bool to true in Client.cpp(for dev branch)

File: README.md [hunk 1]
```diff
@@ -12,27 +12,58 @@ Inspired by the simplicity of [discord.py](https://github.com/Rapptz/discord.py)
...
+        else if (message.content == prefix + "help") { //Prefix with command
+            message.reply("In progress..");
+        }
+        else {
+            //message.reply("command"); //Need to wait for startWith to avoid a response without prefix!
+        };
+    };
+    client.run(token);
+};
+
```
Comment (luccanunes) [inline README.md:61]: Although I like what you went for here, I think we should keep the REAME.md example as simple as we can, and include this in some documentation later.

File: src/Api.cpp [hunk 2]
```diff
@@ -1,41 +1,108 @@
...
+		if(dev){ std::cout << "SENDING: " << id.dump() << "\n\n"; }
+		webSocket.send(id.dump());
+	}
+	void Client::send(const std::string message, const std::string channel_id) {
+		const std::string path = "/channels/" + channel_id + "/messages";
+		json body = { {"content", message} };
+		json res = Api::post(path, cpr::Body{ body.dump() });
+		if(dev){ std::cout << res << std::endl; }
 	}
 }
```
Comment (luccanunes) [inline src/Api.cpp]: Don't fully understand what's going on here. Did you mean to add this changes to Client.cpp instead of Api.cpp?

Comment (HideakiAtsuyo) [reply src/Api.cpp]: No that's an error sorry

File: README.md (hunk 1 above)
Comment (HideakiAtsuyo) [reply README.md:61]: I leave it like that for those who want some more advanced examples but you can modify it if you want to

File: src/Api.cpp (hunk 2 above)
Comment (luccanunes) [reply src/Api.cpp]: No worries

File: README.md (hunk 1 above)
Comment (luccanunes) [reply README.md:61]: I'll add such examples in the documentation website

File: src/Message.cpp [hunk 3]
```diff
@@ -43,4 +43,7 @@ namespace dpp {
 	std::string Message::add_mention(const std::string& emoji) {
 		return Api::put("/channels/" + channel_id + "/messages/" + id + "/reactions/" + emoji + "/@me").dump();
 	}
-}
\ No newline at end of file
+	bool Message::startsWith(const std::string& s) {
+		return content.find(s) == 0;
+	}
```
Comment (luccanunes) [inline src/Message.cpp:52]: This feature has already been added in recent commits at the `main` branch

**Comment by HideakiAtsuyo** [comment]
Checked all your commits and updated everything in the pull request

**Comment by HideakiAtsuyo** [comment]
Checked all your commits and updated everything in the pull request

File: src/Message.cpp (hunk 3 above)
Comment (HideakiAtsuyo) [reply src/Message.cpp:52]: I know i just do that to be sure that everything is correct

**Comment by luccanunes** [comment]
I'm sorry, but I'll have to close this one. There are just a lot of conflicts and not really anything new.
I've already removed all logging from the `main` branch and added it to the `development` one.
If you can, please refrain from making such large pull requests, try to split different topics into different PRs.
Thanks for contributing once again!

**Comment by HideakiAtsuyo** [comment]
> 
> 
> I'm sorry, but I'll have to close this one. There are just a lot of conflicts and not really anything new.
> I've already removed all logging from the `main` branch and added it to the `development` one.
> If you can, please refrain from making such large pull requests, try to split different topics into different PRs.
> Thanks for contributing once again!

the dev bool is better than just remove the std::cout because like this everyone can use it but ok

**Comment by luccanunes** [comment]
I get what you mean but all those logs aren't really necessary to the final user. Specially since all methods (like add_reaction) return a string in json format, with the API response

**Pull request closed without merging**